controllers/user.js

```diff
@@ -689,6 +689,49 @@ async function getRemoveMoreTimePage(req, res) {
   });
 }
 
+function alphabetizeByBroker(data) {
+  return data.sort(function (a, b) {
+    if (a.broker < b.broker) {
+      return -1;
+    }
+    if (a.broker > b.broker) {
+      return 1;
+    }
+    return 0;
```

Comment:
FWIW, I think you can do this w/ `return a.broker.localeCompare(b.broker)` (or close to it).